addons/mail/views/mail_activity_views.xml

```diff
@@ -18,7 +18,9 @@
                     <group>
                         <group>
                             <field name="category" invisible="1"/>
-                            <field name="res_model_id" groups="base.group_no_one"/>
+                            <field name="res_model_id"/>
```

Comment:
By the way shouldn't we remove the model from debug mode it in the list view as well ?

Reply:
est will give me a feedback about that once she has tested it

Reply:
done